open-craft/custom-student-verification
# [BB-3504] Add usage instruction

This PR adds usage instruction for ``custom-student-verification`` pluggable app.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-3504

**Discussions**: N/A

**Dependencies**: None

**Screenshots**: N/A

**Sandbox URL**: N/A

**Merge deadline**: None

**Testing instructions**:

1. Read the usage instruction [here](https://github.com/open-craft/custom-student-verification/blob/shimulch/bb-3504/doc/Usage.md), check for mistakes, improvements.
2. Check that link to the Usage Instruction linked on ``readme.rst``.
**Author notes and concerns**: N/A

**Reviewers**
